addons/hr_timesheet/data/hr_timesheet_demo.xml

```diff
@@ -2818,6 +2818,16 @@
         <field name="company_id" ref="base.main_company"/>
     </record>
 
+    <record id="project_2_task_3_account_analytic_line_28" model="account.analytic.line">
+        <field name="name">Requirements analysis</field>
+        <field name="employee_id" ref="hr.employee_mit"/>
+        <field name="date" eval="(DateTime.now() + relativedelta(months=-2)).strftime('%Y-%m-%d')"/>
+        <field name="unit_amount">4</field>
+        <field name="project_id" ref="project.project_project_2"/>
+        <field name="task_id" ref="project.project_2_task_3"/>
+        <field name="amount">-30.0</field>
```

Comment:
Why is a negative timesheet needed?

Reply:
Here, timesheet is positive unit_amount -> 4. The amount is to track cost without selling time.